coq-core.opam

```diff
@@ -0,0 +1,34 @@
+synopsis: "The Coq Proof Assistant"
+description: """
+Coq is a formal proof management system. It provides
+a formal language to write mathematical definitions, executable
+algorithms and theorems together with an environment for
+semi-interactive development of machine-checked proofs. Typical
+applications include the certification of properties of programming
+languages (e.g. the CompCert compiler certification project, or the
+Bedrock verified low-level programming library), the formalization of
+mathematics (e.g. the full formalization of the Feit-Thompson theorem
+or homotopy type theory) and teaching.
```

Comment:
This description needs to be updated.

Reply:
Indeed, this description should make it clear that this is an internal package not fit for direct installation by most users.

Reply:
Updated, let me know what you think.

Reply:
LGTM.